common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationActivity.java

```diff
@@ -119,6 +119,22 @@ protected void onCreate(final Bundle savedInstanceState) {
             // If activity is killed by the os, savedInstance will be the saved bundle.
             extractState(savedInstanceState);
         }
+
+        if (mAuthorizationAgent == AuthorizationAgent.WEBVIEW) {
+            mAuthorizationStarted = true;
```

Comment:
I don't see this value ever get set to `false` unless `extractState()` is called and that happens _before_ the value is set to `true`

Reply:
Or does it only get set to true if we're using the WebView? In that case, should the variable be renamed to communicate that specifically?

Reply:
Addressed. Thank you, Brian.